### ISSUE TITLE
My Language translations

### ISSUE THREAD

**Issue by femanifest** [400plus/400plus#382]
https://drive.google.com/file/d/0B-3Fgvk6XwFSU1BQbU81RkZQVTA/view?usp=sharing

**Comment by eduperez** [comment]
Just a comment: entries starting with "A_" should exclusively use ASCII characters, as those are rendered by us, and we lack the resources to print anything else.

**Comment by femanifest** [comment]
First now I don't have 400D. But I have ever use 400plus. Really Thanks to You.

And I'm sorry. I can't test this file.

https://drive.google.com/file/d/0B-3Fgvk6XwFSbHZ4dUNyNmFNbVE/view?usp=sharing

I tried one more time. I hope this could be worked

**Comment by eduperez** [comment]
I just tried your file, and I am afraid that it is not working properly: in the camera's firmware, Asian fonts are not monospaced, and we align the menus counting the characters in each label; when the characters do not have all the same width, we cannot align the labels, and the menu looks wrong.

We already detected this for the Chinese translation (see issue #367), and we could not find a solution. There is nothing wrong with what you did, and it is a pity that it cannot be used.

Thanks!
